examples/python-cicd-with-cloudbuilder/python-cloud-builder/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# This Dockerfile is used to build a custom cloud builder, shown in step 3 of README.md
+# It pulls the public python 3 image and then installs virtualenv so that a virtual environment can
```

Comment:
public python 3 cloud builder container image? Or just container image? Just want to make sure I'm clear. I'm not super familiar with containers.

Reply:
It pulls the python:3 image from Dockerhub (https://hub.docker.com/_/python). If you don't specify a URL in FROM it will default to hub.docker.com. 

I clarified in the comments as well.